src/test/rustdoc/synthetic_auto/no-redundancy.rs

```diff
@@ -20,7 +20,7 @@ where
 
 // @has no_redundancy/struct.Outer.html
 // @has - '//*[@id="synthetic-implementations-list"]/*[@class="impl"]/*/code' "impl<T> Send for \
-// Outer<T> where T: Copy + Send"
+// Outer<T> where T: Send + Copy"
```

Comment:
The `ptr::Shared` removal makes this test fail unless I make this change. I have no idea why.

Reply:
Filed https://github.com/rust-lang/rust/issues/49123